### DISCUSSION TITLE
Labels and process improvement

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#4445] [General]
Hey, since last year, we've added a bunch of labels. 

I believe there is always room to experiment, improve and speed up the process 

Currently there are few new labels which I encourage to use 
## Issues
- `design-request` - is where we spotted during redesign or new feature would need request design drop from designer
- [`navbar`, `item`, `landing`, .. ] - name component where things is needed to be done, which section, some ppl specializes much better in some
- `mobile` - responsive design is obviously challenging
- `chief` - means that issue carry sub-issues, it's helps tracking, something like milestone styled, yet we don't need to fiddle with milestones/projects within github, for now
- `multi-chain` - involve more indexer/cross chain knowledge
- `hackathon` - it's intended for hackathons participants, but could be done outside hackathon too! 

## Pull-requests
To speed up flow on merging, we try to always respect green check from CI/CD tests running, other than that, we add some human tags to respect approval
- `code-lgtm` - means code looks ok and no major reservation to code change and it would work
- `works-for-me` - means that someone tested desired functionality and did not spot that something else has broke with this PR
- `visual-ok` - means, that PR follows design from issue/figma file

## Which labels to add?

Now the question in the room is, which label we should have to align our development process to be better? I'm always trying to be open-minded to embrace new labels, if they add quality, streamline and make it easy for everyone else our developer process, it would be more than welcome.

**Comment by yangwao** [comment]
One thing @preschian mentioned that we should require deepscan and deepsource checks on merging to avoid technical debt, wyt?

![image](https://user-images.githubusercontent.com/5887929/211199384-23391f3b-7737-483e-8b44-31d12c1409b4.png)

**Reply by preschian** [reply]
yes please, I think we need revisit our linter also in the future

**Comment by yangwao** [comment]
Other good hint is I'll take some time off for myself and focus on strategic stuff, hence we will have on board more folks and some things can work without me now flawlessly (which I'm superior grateful for 😌)

I'll take time windows to review PRs and help answer stuff on Mondays and Thursdays between 12pm-1pm UTC+1.

**Comment by roiLeo** [comment]
> ## Which labels to add?

- Difficulty level: **Beginner**; **Medium**; **Expert** 
- needs more infos / need discussion
- parent task / sub task
- dependencies
